From: Qingqing <wangyiqing@kylinos.cn>
Date: Mon, 20 Jul 2026 08:59:06 +0000
Subject: Translated using Weblate (Indonesian)

Currently translated at 87.5% (63 of 72 strings)

Translation: openkylin-huanghe/peony-extension-computer-view
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/peony-extension-computer-view/id/
---
 .../translations/peony-engrampa-menu_id.ts         |  10 +-
 .../peony-extension-computer-view_id.ts            | 126 +++++++++++----------
 2 files changed, 69 insertions(+), 67 deletions(-)

diff --git a/peony-engrampa-menu-plugin/translations/peony-engrampa-menu_id.ts b/peony-engrampa-menu-plugin/translations/peony-engrampa-menu_id.ts
index 11ebc7e..d7810e8 100644
--- a/peony-engrampa-menu-plugin/translations/peony-engrampa-menu_id.ts
+++ b/peony-engrampa-menu-plugin/translations/peony-engrampa-menu_id.ts
@@ -34,12 +34,12 @@
     <message>
         <location filename="../engrampa-menu-plugin.h" line="40"/>
         <source>Peony-Qt engrampa Extension</source>
-        <translation type="unfinished"></translation>
+        <translation>Plugin Kompresi</translation>
     </message>
     <message>
         <location filename="../engrampa-menu-plugin.h" line="41"/>
         <source>Engrampa Menu Extension</source>
-        <translation type="unfinished"></translation>
+        <translation>Ekstensi Menu Arsip</translation>
     </message>
     <message>
         <source>engrampa Menu Extension</source>
@@ -52,17 +52,17 @@
     <message>
         <location filename="../engrampa-menu-plugin.cpp" line="74"/>
         <source>Compress</source>
-        <translation type="unfinished"></translation>
+        <translation>Kompres</translation>
     </message>
     <message>
         <location filename="../engrampa-menu-plugin.cpp" line="90"/>
         <source>Uncompress to current path</source>
-        <translation type="unfinished"></translation>
+        <translation>Ekstrak di Sini</translation>
     </message>
     <message>
         <location filename="../engrampa-menu-plugin.cpp" line="101"/>
         <source>Uncompress to specific path...</source>
-        <translation type="unfinished"></translation>
+        <translation>Ekstrak ke...</translation>
     </message>
 </context>
 </TS>
diff --git a/peony-extension-computer-view/peony-extension-computer-view_id.ts b/peony-extension-computer-view/peony-extension-computer-view_id.ts
index 1902825..a4f11b3 100644
--- a/peony-extension-computer-view/peony-extension-computer-view_id.ts
+++ b/peony-extension-computer-view/peony-extension-computer-view_id.ts
@@ -6,7 +6,7 @@
     <message>
         <location filename="computer-view/computer-item-delegate.cpp" line="174"/>
         <source>You should mount volume first</source>
-        <translation type="unfinished"></translation>
+        <translation>Perlu Mengaitkan Partisi Terlebih Dahulu</translation>
     </message>
 </context>
 <context>
@@ -14,7 +14,7 @@
     <message>
         <location filename="computer-view/computer-network-item.cpp" line="45"/>
         <source>Network Neighborhood</source>
-        <translation type="unfinished"></translation>
+        <translation>Lingkungan Jaringan</translation>
     </message>
 </context>
 <context>
@@ -22,7 +22,7 @@
     <message>
         <location filename="computer-view/computer-remote-volume-item.cpp" line="84"/>
         <source>Remote</source>
-        <translation type="unfinished"></translation>
+        <translation>Direktori Jarak Jauh</translation>
     </message>
 </context>
 <context>
@@ -34,7 +34,7 @@
     <message>
         <location filename="computer-view/computer-user-share-item.cpp" line="45"/>
         <source>Data</source>
-        <translation type="unfinished"></translation>
+        <translation>Disk Data</translation>
     </message>
 </context>
 <context>
@@ -42,13 +42,13 @@
     <message>
         <location filename="computer-view/computer-volume-item.cpp" line="69"/>
         <source>Volume</source>
-        <translation type="unfinished"></translation>
+        <translation>Partisi Lokal</translation>
     </message>
     <message>
         <location filename="computer-view/computer-volume-item.cpp" line="82"/>
         <location filename="computer-view/computer-volume-item.cpp" line="146"/>
         <source>System Disk</source>
-        <translation type="unfinished"></translation>
+        <translation>Disk Sistem</translation>
     </message>
     <message>
         <source>File System</source>
@@ -57,7 +57,7 @@
     <message>
         <location filename="computer-view/computer-volume-item.cpp" line="210"/>
         <source>Data</source>
-        <translation type="unfinished"></translation>
+        <translation>Disk Data</translation>
     </message>
 </context>
 <context>
@@ -65,7 +65,7 @@
     <message>
         <location filename="computer-view-intel/intel-computer-item-delegate.cpp" line="150"/>
         <source>You should mount volume first</source>
-        <translation type="unfinished"></translation>
+        <translation>Perlu Mengaitkan Partisi Terlebih Dahulu</translation>
     </message>
 </context>
 <context>
@@ -73,7 +73,7 @@
     <message>
         <location filename="computer-view-intel/intel-computer-network-item.cpp" line="47"/>
         <source>Network Neighborhood</source>
-        <translation type="unfinished"></translation>
+        <translation>Lingkungan Jaringan</translation>
     </message>
 </context>
 <context>
@@ -81,7 +81,7 @@
     <message>
         <location filename="computer-view-intel/intel-computer-remote-volume-item.cpp" line="56"/>
         <source>Remote</source>
-        <translation type="unfinished"></translation>
+        <translation>Direktori Jarak Jauh</translation>
     </message>
 </context>
 <context>
@@ -89,28 +89,28 @@
     <message>
         <location filename="computer-view-intel/intel-computer-view-container.cpp" line="105"/>
         <source>Connect a server</source>
-        <translation type="unfinished"></translation>
+        <translation>Hubungkan ke Server Jarak Jauh</translation>
     </message>
     <message>
         <location filename="computer-view-intel/intel-computer-view-container.cpp" line="139"/>
         <source>Unmount</source>
-        <translation type="unfinished"></translation>
+        <translation>Lepaskan</translation>
     </message>
     <message>
         <location filename="computer-view-intel/intel-computer-view-container.cpp" line="145"/>
         <source>Eject</source>
-        <translation type="unfinished"></translation>
+        <translation>Lepaskan</translation>
     </message>
     <message>
         <location filename="computer-view-intel/intel-computer-view-container.cpp" line="155"/>
         <location filename="computer-view-intel/intel-computer-view-container.cpp" line="171"/>
         <source>Property</source>
-        <translation type="unfinished"></translation>
+        <translation>Properti</translation>
     </message>
     <message>
         <location filename="computer-view-intel/intel-computer-view-container.cpp" line="157"/>
         <source>You have to mount this volume first</source>
-        <translation type="unfinished"></translation>
+        <translation>Perlu memuat partisi terlebih dahulu</translation>
     </message>
 </context>
 <context>
@@ -118,18 +118,18 @@
     <message>
         <location filename="computer-view-intel/intel-computer-volume-item.cpp" line="39"/>
         <source>Volume</source>
-        <translation type="unfinished"></translation>
+        <translation>Partisi Lokal</translation>
     </message>
     <message>
         <location filename="computer-view-intel/intel-computer-volume-item.cpp" line="47"/>
         <location filename="computer-view-intel/intel-computer-volume-item.cpp" line="109"/>
         <source>System Disk</source>
-        <translation type="unfinished"></translation>
+        <translation>Disk Sistem</translation>
     </message>
     <message>
         <location filename="computer-view-intel/intel-computer-volume-item.cpp" line="99"/>
         <source>User Disk</source>
-        <translation type="unfinished"></translation>
+        <translation>Disk Pengguna</translation>
     </message>
 </context>
 <context>
@@ -137,47 +137,47 @@
     <message>
         <location filename="login-remote-filesystem.ui" line="41"/>
         <source>Connect to Sever</source>
-        <translation type="unfinished"></translation>
+        <translation>Hubungkan ke Server</translation>
     </message>
     <message>
         <location filename="login-remote-filesystem.ui" line="89"/>
         <source>server information</source>
-        <translation type="unfinished"></translation>
+        <translation>Informasi Server</translation>
     </message>
     <message>
         <location filename="login-remote-filesystem.ui" line="111"/>
         <source>user information</source>
-        <translation type="unfinished"></translation>
+        <translation>Informasi Pengguna</translation>
     </message>
     <message>
         <location filename="login-remote-filesystem.ui" line="141"/>
         <source>tag</source>
-        <translation type="unfinished"></translation>
+        <translation>Tab</translation>
     </message>
     <message>
         <location filename="login-remote-filesystem.ui" line="196"/>
         <source>user</source>
-        <translation type="unfinished"></translation>
+        <translation>Nama Pengguna</translation>
     </message>
     <message>
         <location filename="login-remote-filesystem.ui" line="252"/>
         <source>password</source>
-        <translation type="unfinished"></translation>
+        <translation>Kata Sandi</translation>
     </message>
     <message>
         <location filename="login-remote-filesystem.ui" line="308"/>
         <source>protocol</source>
-        <translation type="unfinished"></translation>
+        <translation>Protokol</translation>
     </message>
     <message>
         <location filename="login-remote-filesystem.ui" line="351"/>
         <source>server</source>
-        <translation type="unfinished"></translation>
+        <translation>Alamat Server</translation>
     </message>
     <message>
         <location filename="login-remote-filesystem.ui" line="382"/>
         <source>directory</source>
-        <translation type="unfinished"></translation>
+        <translation>Direktori</translation>
     </message>
     <message>
         <location filename="login-remote-filesystem.ui" line="411"/>
@@ -197,7 +197,7 @@
     <message>
         <location filename="login-remote-filesystem.ui" line="476"/>
         <source>port</source>
-        <translation type="unfinished"></translation>
+        <translation>Port</translation>
     </message>
     <message>
         <location filename="login-remote-filesystem.ui" line="508"/>
@@ -232,12 +232,12 @@
     <message>
         <location filename="login-remote-filesystem.ui" line="592"/>
         <source>ok</source>
-        <translation type="unfinished"></translation>
+        <translation>OK</translation>
     </message>
     <message>
         <location filename="login-remote-filesystem.ui" line="623"/>
         <source>cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Batal</translation>
     </message>
 </context>
 <context>
@@ -245,7 +245,7 @@
     <message>
         <location filename="computer-view-container.cpp" line="584"/>
         <source>Connect a server</source>
-        <translation type="unfinished"></translation>
+        <translation>Hubungkan ke Server Jarak Jauh</translation>
     </message>
     <message>
         <source>sftp://, etc...</source>
@@ -262,12 +262,12 @@
     <message>
         <location filename="computer-view-container.cpp" line="638"/>
         <source>Unmount</source>
-        <translation type="unfinished"></translation>
+        <translation>Lepaskan</translation>
     </message>
     <message>
         <location filename="computer-view-container.cpp" line="652"/>
         <source>Eject</source>
-        <translation type="unfinished"></translation>
+        <translation>Lepaskan</translation>
     </message>
     <message>
         <source>format</source>
@@ -278,28 +278,28 @@
         <location filename="computer-view-container.cpp" line="722"/>
         <location filename="computer-view-container.cpp" line="735"/>
         <source>Format</source>
-        <translation type="unfinished"></translation>
+        <translation>Format</translation>
     </message>
     <message>
         <location filename="computer-view-container.cpp" line="758"/>
         <location filename="computer-view-container.cpp" line="803"/>
         <source>Property</source>
-        <translation type="unfinished"></translation>
+        <translation>Properti</translation>
     </message>
     <message>
         <location filename="computer-view-container.cpp" line="760"/>
         <source>You have to mount this volume first</source>
-        <translation type="unfinished"></translation>
+        <translation>Perlu memuat partisi terlebih dahulu</translation>
     </message>
     <message>
         <location filename="computer-view-container.cpp" line="783"/>
         <source>Open In New Window</source>
-        <translation type="unfinished"></translation>
+        <translation>Buka di Jendela Baru</translation>
     </message>
     <message>
         <location filename="computer-view-container.cpp" line="793"/>
         <source>Open In New Tab</source>
-        <translation type="unfinished"></translation>
+        <translation>Buka di Tab Baru</translation>
     </message>
 </context>
 <context>
@@ -308,34 +308,34 @@
         <location filename="../peony-drive-rename/drive-rename.cpp" line="206"/>
         <location filename="../peony-drive-rename/drive-rename.cpp" line="218"/>
         <source>Rename</source>
-        <translation type="unfinished"></translation>
+        <translation>Ganti Nama</translation>
     </message>
     <message>
         <location filename="../peony-drive-rename/drive-rename.cpp" line="218"/>
         <source>Device name:</source>
-        <translation type="unfinished"></translation>
+        <translation>Nama Perangkat:</translation>
     </message>
     <message>
         <location filename="../peony-drive-rename/drive-rename.cpp" line="222"/>
         <location filename="../peony-drive-rename/drive-rename.cpp" line="230"/>
         <location filename="../peony-drive-rename/drive-rename.cpp" line="237"/>
         <source>Warning</source>
-        <translation type="unfinished"></translation>
+        <translation>Peringatan</translation>
     </message>
     <message>
         <location filename="../peony-drive-rename/drive-rename.cpp" line="222"/>
         <source>Renaming cannot start with a decimal point, Please re-enter!</source>
-        <translation type="unfinished"></translation>
+        <translation>Karakter pertama tidak boleh berupa titik saat mengganti nama, silakan masukkan ulang!</translation>
     </message>
     <message>
         <location filename="../peony-drive-rename/drive-rename.cpp" line="230"/>
         <source>The device name exceeds the character limit, rename failed!</source>
-        <translation type="unfinished"></translation>
+        <translation>Nama perangkat melebihi batas karakter, gagal mengganti nama!</translation>
     </message>
     <message>
         <location filename="../peony-drive-rename/drive-rename.cpp" line="237"/>
         <source>Renaming will unmount the device. Do you want to continue?</source>
-        <translation type="unfinished"></translation>
+        <translation>Mengganti nama akan melepaskan perangkat, lanjutkan?</translation>
     </message>
 </context>
 <context>
@@ -343,19 +343,19 @@
     <message>
         <location filename="peony-computer-view-plugin.h" line="42"/>
         <source>Computer View</source>
-        <translation type="unfinished"></translation>
+        <translation>Tampilan Komputer</translation>
     </message>
     <message>
         <location filename="peony-computer-view-plugin.h" line="43"/>
         <source>Show drives, network and personal directories</source>
         <oldsource>Show drives, network and personal directories.</oldsource>
-        <translation type="unfinished"></translation>
+        <translation>Tampilkan partisi perangkat, direktori jaringan, dan direktori pribadi</translation>
     </message>
     <message>
         <location filename="computer-view-intel/intel-computer-volume-item.cpp" line="517"/>
         <location filename="computer-view/computer-volume-item.cpp" line="887"/>
         <source>One or more programs prevented the unmount operation.</source>
-        <translation type="unfinished"></translation>
+        <translation>Satu atau beberapa program menghalangi operasi pelepasan.</translation>
     </message>
     <message>
         <location filename="computer-view-intel/intel-computer-volume-item.cpp" line="519"/>
@@ -364,51 +364,53 @@
         <location filename="computer-view/computer-volume-item.cpp" line="892"/>
         <location filename="computer-view/computer-volume-item.cpp" line="896"/>
         <source>Unmount failed</source>
-        <translation type="unfinished"></translation>
+        <translation>Pelepasan Gagal</translation>
     </message>
     <message>
         <location filename="computer-view-intel/intel-computer-volume-item.cpp" line="520"/>
         <source>Error: %1
 Do you want to unmount forcely?</source>
-        <translation type="unfinished"></translation>
+        <translation>Kesalahan: %1
+Apakah akan dipaksakan untuk dilepas kaitnya?</translation>
     </message>
     <message>
         <location filename="computer-view/computer-volume-item.cpp" line="363"/>
         <location filename="computer-view/computer-volume-item.cpp" line="461"/>
         <location filename="computer-view/computer-volume-item.cpp" line="1301"/>
         <source>It need to synchronize before operating the device,place wait!</source>
-        <translation type="unfinished"></translation>
+        <translation>Perlu menyinkronkan data sebelum mengoperasikan perangkat, harap tunggu!</translation>
     </message>
     <message>
         <location filename="computer-view/computer-volume-item.cpp" line="844"/>
         <source>The device has been mount successfully!</source>
-        <translation type="unfinished"></translation>
+        <translation>Perangkat berhasil dimuat!</translation>
     </message>
     <message>
         <location filename="computer-view/computer-volume-item.cpp" line="865"/>
         <location filename="computer-view/computer-volume-item.cpp" line="915"/>
         <location filename="computer-view/computer-volume-item.cpp" line="956"/>
         <source>Data synchronization is complete,the device has been unmount successfully!</source>
-        <translation type="unfinished"></translation>
+        <translation>Sinkronisasi data selesai, perangkat berhasil dilepaskan!</translation>
     </message>
     <message>
         <location filename="computer-view/computer-volume-item.cpp" line="888"/>
         <location filename="computer-view/computer-volume-item.cpp" line="896"/>
         <source>Error: %1
 </source>
-        <translation type="unfinished"></translation>
+        <translation>Kesalahan: %1\n
+</translation>
     </message>
     <message>
         <location filename="computer-view/computer-volume-item.cpp" line="924"/>
         <location filename="computer-view/computer-volume-item.cpp" line="950"/>
         <source>Eject device failed, the reason may be that the device has been removed, etc.</source>
-        <translation type="unfinished"></translation>
+        <translation>Gagal melepas perangkat, mungkin perangkat telah dicabut.</translation>
     </message>
     <message>
         <location filename="computer-view-intel/intel-computer-volume-item.cpp" line="527"/>
         <location filename="computer-view/computer-volume-item.cpp" line="891"/>
         <source>Unable to unmount it, you may need to close some programs, such as: GParted etc.</source>
-        <translation type="unfinished"></translation>
+        <translation>Tidak dapat melepas kait, Anda mungkin perlu menutup beberapa program terlebih dahulu, seperti editor partisi dll.</translation>
     </message>
     <message>
         <location filename="computer-view-intel/intel-computer-volume-item.cpp" line="529"/>
@@ -421,38 +423,38 @@ Do you want to unmount forcely?</source>
         <location filename="computer-view/computer-volume-item.cpp" line="926"/>
         <location filename="computer-view/computer-volume-item.cpp" line="952"/>
         <source>Eject failed</source>
-        <translation type="unfinished"></translation>
+        <translation>Pelepasan Gagal</translation>
     </message>
     <message>
         <location filename="computer-view-intel/intel-computer-volume-item.cpp" line="554"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Batal</translation>
     </message>
     <message>
         <location filename="computer-view-intel/intel-computer-volume-item.cpp" line="555"/>
         <source>Eject Anyway</source>
-        <translation type="unfinished"></translation>
+        <translation>Lepaskan Apa Pun</translation>
     </message>
     <message>
         <location filename="../peony-drive-rename/drive-rename.cpp" line="405"/>
         <source>Warning</source>
-        <translation type="unfinished"></translation>
+        <translation>Peringatan</translation>
     </message>
     <message>
         <location filename="../peony-drive-rename/drive-rename.cpp" line="405"/>
         <source>The device may not support the rename operation, rename failed!</source>
-        <translation type="unfinished"></translation>
+        <translation>Perangkat mungkin tidak mendukung operasi penggantian nama, penggantian nama gagal!</translation>
     </message>
     <message>
         <location filename="computer-view-container.cpp" line="75"/>
         <source>Message recipient disconnected from message bus without replying!</source>
-        <translation type="unfinished"></translation>
+        <translation>Penerima pesan terputus dari bus pesan tanpa balasan!</translation>
     </message>
     <message>
         <location filename="computer-view-container.cpp" line="77"/>
         <location filename="computer-view-intel/intel-computer-view-container.cpp" line="59"/>
         <source>log remote error</source>
-        <translation type="unfinished"></translation>
+        <translation>Log Kesalahan Jarak Jauh</translation>
     </message>
 </context>
 </TS>
